crt_portal/cts_forms/tests/test_forms.py

```diff
@@ -464,6 +464,20 @@ def test_post_with_ids(self):
             self.assertEquals(last_activity.description, 'Updated from "None" to "DELETE_USER"')
             self.assertEquals(last_activity.actor, user)
 
+    def test_post_with_section_status_and_assignee(self):
+        ids = [report.id for report in self.reports[3:5]]
+        user = User.objects.get(username='DELETE_USER')
```

Comment:
Small opportunity here to use `self.user` instead of fetching from database again.